psalm.xml

```diff
@@ -29,4 +29,11 @@
         <function name="var_dump"/>
         <function name="print_r"/>
     </forbiddenFunctions>
+    <issueHandlers>
+        <UndefinedInterfaceMethod>
+            <errorLevel type="suppress">
+                <file name="src/Form/Type/CustomDiscountCollectionType.php"/>
+            </errorLevel>
+        </UndefinedInterfaceMethod>
+    </issueHandlers>
```

Comment:
Not a fan. Should be possible to avoid with asserts I guess

Reply:
I'm not the fan either :| The problem is, the `AdjustableInterface` from Sylius **for some reason** does not contain `getAdjustmentsRecursively` method - it's provided separately for `OrderInterface` and `OrderItemInterface`. I even have a check on the function level `Assert::isInstanceOfAny($adjustable, [OrderInterface::class, OrderItemInterface::class]);`, but it's not recognizable by psalm... do you maybe know an annotation I can use for the getter function to tell psalm what to expect?

Reply:
OK, I figured it out :smile: 🚀